service/src/main/java/uk/gov/hmcts/reform/fpl/service/CaseManagementOrderService.java

```diff
@@ -64,6 +66,14 @@ public OrderAction removeDocumentFromOrderAction(OrderAction orderAction) {
         return orderAction.toBuilder().document(null).build();
     }
 
+    public boolean isHearingDateInFuture(CaseData caseData) {
```

Comment:
Shouldn't we be a bit more specific what hearing date? The logic below takes cmoToAction. 

Partially you can ge this from CMOService, but still might not be clear as there are multiple CMOs. 

Or do you think it's clear enough?

Reply:
I think for now this is fine. I am desparate to refactor the hearingBooking / draftCMO stuff.